client/src/Components/Overview.jsx

```diff
@@ -119,75 +56,6 @@ class Overview extends React.Component{
            <hr></hr> </div>
         : null }  
```

Comment:
i like this use of a ternary operator; it would be especially useful in modals.  I may change mine to this :)